dev/tasks/tasks.yml

```diff
@@ -1185,13 +1185,22 @@ tasks:
       image: ubuntu-r-only-r
 
   test-r-offline-minimal:
-      ci: azure
-      template: r/azure.linux.yml
-      params:
-        r_org: rocker
-        r_image: r-base
-        r_tag: latest
-        flags: '-e TEST_OFFLINE_BUILD=true'
+    ci: azure
+    template: r/azure.linux.yml
+    params:
+      r_org: rocker
+      r_image: r-base
+      r_tag: latest
+      flags: '-e TEST_OFFLINE_BUILD=true'
+
+  test-r-dev-duckdb:
+    ci: github
+    template: docker-tests/github.linux.yml
+    params:
+      env:
+        R_DUCKDB_DEV: TRUE
+      flags: '-e ARROW_SOURCE_HOME="/arrow" -e FORCE_BUNDLED_BUILD=TRUE -e LIBARROW_BUILD=TRUE'
```

Comment:
Why are these flags necessary? That's the default for `ARROW_SOURCE_HOME` and `LIBARROW_BUILD` right? Or maybe the question is: why is this using a different `template`?